src/main/java/de/zalando/ep/zalenium/proxy/DockerSeleniumRemoteProxy.java

```diff
@@ -66,7 +67,7 @@
     @VisibleForTesting
     public static final String ZALENIUM_MAX_TEST_SESSIONS = "ZALENIUM_MAX_TEST_SESSIONS";
     @VisibleForTesting
-    public static final long DEFAULT_MAX_TEST_IDLE_TIME_SECS = 90L;
+    public static final long DEFAULT_MAX_TEST_IDLE_TIME_SECS = 300L;
```

Comment:
This value can be modified through an environment variable, right? Why do we need to change it here?

Reply:
Yes, we should remove it from here.

Reply:
Sorry, I just realize that that timeout can't  be modified using environment variables, you only can change it using the request capabilities.

Reply:
Yes, you are right. I confused it with other value. 
But then, the better approach would be to make this value configurable, right?

Reply:
done